client/package.json

```diff
@@ -40,6 +40,7 @@
 		"@automattic/happychat-connection": "workspace:^",
 		"@automattic/help-center": "workspace:^",
 		"@automattic/i18n-utils": "workspace:^",
+		"@automattic/isolated-block-editor": "2.22.0",
```

Comment:
Is there a need to pin it like that, or could we make it less specific with a `^`?

Reply:
`^` doesn't work, since 0.23.0 updates to React 18 and so is incompatible. But I changed it to 0.22.x.